.github/workflows/pants.yaml

```diff
@@ -57,9 +56,3 @@ jobs:
     - name: Test
       run: |
         ./pants test ::
-    - name: Upload pants log
-      uses: actions/upload-artifact@v2
-      with:
-        name: pants-log
-        path: .pants.d/pants.log
-      if: always()  # We want the log even on failures.
```

Comment:
Why remove this?

Reply:
Because the file didn't exist and so the build was failing. Can re-add later if need be but I want to get CI working first.